### PR TITLE
LPS-151430 [CP 2.0] Create Provisioning role

### PR DESCRIPTION
**Pull request by igordmf**

### CODE REVIEW

**Review by claricebrasil — APPROVED** [review]
Good job!

**Review by thaynam — APPROVED** [review]

**Review by joaosvassis — APPROVED** [review]

**Comment by jairmedeiros** [comment]
@joseabelenda

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/AnalyticsCloud/index.js [hunk 1]
```diff
@@ -114,7 +114,7 @@ const ActivationStatusAnalyticsCloud = ({
 			title: 'Analytics Cloud Activation',
 		},
 		[STATUS_TAG_TYPE_NAMES.inProgress]: {
-			dropdownIcon: userAccount.isStaff && (
+			dropdownIcon: userAccount.isStaff && userAccount.isProvisioning && (
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/AnalyticsCloud/index.js:117]: I guess the stories need to be staff or provisioning user role.

```suggestion
			dropdownIcon: (userAccount.isStaff || userAccount.isProvisioning) && (
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/DXPCloud/index.js [hunk 2]
```diff
@@ -148,7 +148,7 @@ const ActivationStatusDXPCloud = ({
 			title: 'Activation Status',
 		},
 		[STATUS_TAG_TYPE_NAMES.inProgress]: {
-			dropdownIcon: userAccount.isStaff && (
+			dropdownIcon: userAccount.isStaff && userAccount.isProvisioning && (
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/DXPCloud/index.js:151]: I guess the stories need to be staff or provisioning user role.

```suggestion
			dropdownIcon: (userAccount.isStaff || userAccount.isProvisioning) && (
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward

**Comment by joseabelenda** [comment]
@nNilton

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**